keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,162 @@
+1. When should the Kubernetes documentation host third party documents that we are not the source/authority of?
+1. How does the Kubernetes project handle third party content that is not kept up to date or hosts?
+1. Can feature owners flag when third party content is *required*, as opposed to preferable or common?
+1. Who decides when to include third-party content?
+1. What standard of quality and review must be met before docs include third-party content?
+1. To what extent should SIG Docs advocate for third-party content providers to host their own content, or decline to host third-party content altogether?
+
+### Goals
+
+- Policy on including content about third party products in Kubernetes web documentation.
+- Published guidelines for contributors and reviewers that clarify requirements for third party content and allow documentation reviewers to reject advertising.
+
+
+### Non-Goals
+
+- Outright removal of all content relating to vendors and projects outside the CNCF ecosystem.
+
+## Proposal
+
+TBD. Detailed policies will come out of the discussion around this KEP.
+
+### User Stories
+
+
+#### Story 1 (fictional)
+
+Alice works for ACME, Inc and wants to gain visibility for ACME Cloud Services, which has just launched a managed Kubernetes cluster product. Alice drafts a change to a concept page so that that it mentions ACME Cloud Services’ Kubernetes product, and submits a pull request.
+Bob is a documentation approver. Bob explains that Alice’s proposed change does not meet community standards, because it is functionally an advertisement.
+
+#### Story 2 (fictional)
+
+Charlie uses Linux, specifically Ubuntu. Charlie notices that the page about installing `kubethingy` has instructions for installing `kubethingy` on Windows and on CentOS/RHEL but not on Ubuntu. Charlie reads the guidelines on content and sees that this kind of change is acceptable (Ubuntu is one of the most popular Linux distributions, and `kubethingy` documentation is acceptable as it is already documented). Charlie drafts a change and submits a pull request.
+
+#### Story 3 (actual)
+
+Rafael wanted to share a Kubernetes course from an online education provider. He submitted [PR #15962](https://github.com/kubernetes/website/pull/15962) to add the course to [Overview of Kubernetes Online Training](https://kubernetes.io/docs/tutorials/online-training/overview/). The PR was not approved because SIG Docs didn’t want to add a link to third-party content over which SIG Docs have no control.
+
+#### Story 4 (actual)
+
+[PR #16203](https://github.com/kubernetes/website/pull/16203) removes Stackdriver and Elasticsearch vendor content. Since logging falls into the external add-ons category, SIG Docs decided to remove this vendor-specific content. That , which has not been meaningfully updated in three years. SIG Docs had buy-in from SIG Instrumentation Bugs for removal; however that PR was held pending the outcome of this KEP.
```

Comment:
`. That , which` -> `, which`?

Reply:
"Since logging falls into the external add-ons category, SIG Docs decided to remove this vendor-specific content that has not been meaningfully updated in three years."

Reply:
Updated